akka-typed/src/main/scala/akka/typed/ActorContext.scala

```diff
@@ -62,12 +62,12 @@ trait ActorContext[T] {
    * Create a child Actor from the given [[Props]] under a randomly chosen name.
    * It is good practice to name Actors wherever practical.
    */
-  def spawnAnonymous[U](props: Props[U]): ActorRef[U]
+  def spawnAnonymous[U](behavior: Behavior[U], dispatcher: DispatcherSelector = DispatcherDefault, mailboxCapacity: Int = 1000000): ActorRef[U]
```

Comment:
Yes, exactly, this was one of the counterarguments, and it is valid. However, given the long history of this API and our experience with it and the new tendency of rather removing stuff than adding it, plus the unlikeliness that we want to touch this in the next 2 years, I would say that this risk is rather low. We should avoid default arguments though.